ConsoleApp/Program.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System;
+using System.Threading;
+using System.Collections.Generic;
+using System.Linq;
+namespace ConsoleApp;
+
+    class Game
+    {
+        static readonly int x = 50;

```

Comment:
code convention violated, should be
```suggestion
        private static readonly int _x = 50;
```

but better
```suggestion
        private const int X = 50;
```